### PR TITLE
Add nekoray-bin

### PR DESCRIPTION
**Pull request by Molyuu**
Add nekoray-bin.
I'm also maintaining the package on [AUR](https://aur.archlinux.org/packages/nekoray-bin)

### CODE REVIEW

**Comment by hubutui** [comment]
building from the source is preferred if this pkg is opensource.

File: archlinuxcn/nekoray-bin/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Maintainer: Molyuu <zhangjtroger@gmail.com>
+pkgname='nekoray-bin'
+pkgver=2.14
+_releasedate=2023-02-04
+pkgrel=1
+pkgdesc='Qt based cross-platform GUI proxy configuration manager (backend: v2ray / sing-box)'
+arch=('x86_64')
+url='https://github.com/MatsuriDayo/nekoray'
+license=('GPL 3.0')
```
Comment (hubutui) [inline archlinuxcn/nekoray-bin/PKGBUILD:9]: This should be `GPL3`?

Comment (Molyuu) [reply archlinuxcn/nekoray-bin/PKGBUILD:9]: Ah,yes.Thanks for pointing it out.

**Comment by Molyuu** [comment]
> building from the source is preferred if this pkg is opensource.

Thanks for your fast reply.I will close this pull request and open a new one later.

**Pull request closed without merging**